### PR TITLE
Update snitcher

### PR DESCRIPTION
**Pull request by philipktlin**
Update snitcher docker files and related scripts

### CODE REVIEW

**Comment by arsing** [comment]
Don't check in a binary (vcruntime140.dll)

**Comment by philipktlin** [comment]
> Don't check in a binary (vcruntime140.dll)

Updated to copy it from Windows base image.

File: tools/snitch/snitcher/docker/linux/amd64/Dockerfile [hunk 1]
```diff
@@ -1,21 +1,12 @@
...
-RUN cargo build --release
-
-# Build real container
-
-FROM alpine:latest
-RUN apk --no-cache add ca-certificates
-COPY --from=builder /home/rust/src/target/x86_64-unknown-linux-musl/release/snitcher \
-    /usr/local/bin/
-
+FROM ubuntu:18.04
```
Comment (arsing) [inline tools/snitch/snitcher/docker/linux/amd64/Dockerfile]: Why do you need this change?

File: tools/snitch/snitcher/docker/linux/amd64/Dockerfile [hunk 2]
```diff
@@ -1,21 +1,12 @@
...
+
+RUN apt-get update -y && \
+    apt-get install -y apt-utils ca-certificates curl gpg && \
+    curl https://ftp-master.debian.org/keys/archive-key-9-security.asc | gpg --dearmor > archive-key-9-security.gpg && \
+    cp ./archive-key-9-security.gpg /etc/apt/trusted.gpg.d/ &&\
+    echo "deb http://security.debian.org/debian-security stretch/updates main" >> /etc/apt/sources.list && \
+    apt-get update -y && \
+    apt-get install -y libssl1.0.2
```
Comment (arsing) [inline tools/snitch/snitcher/docker/linux/amd64/Dockerfile]: Why do you need an older openssl?

Comment (philipktlin) [reply tools/snitch/snitcher/docker/linux/amd64/Dockerfile]: This is one of the dependencies snitcher required.  And the new version is not working for Ubuntu 18.04.

File: tools/snitch/snitcher/docker/linux/amd64/Dockerfile (hunk 1 above)
Comment (philipktlin) [reply tools/snitch/snitcher/docker/linux/amd64/Dockerfile]: It has error from 1 of build steps.  Let me run again and copy the error here.

File: tools/snitch/snitcher/docker/linux/amd64/Dockerfile (hunk 2 above)
Comment (philipktlin) [reply tools/snitch/snitcher/docker/linux/amd64/Dockerfile]: @darobs would you please help to explain the problem that libssl1.0.2 needed to use on ubuntu:18.04?  thanks.

File: tools/snitch/snitcher/docker/linux/amd64/Dockerfile (hunk 1 above)
Comment (philipktlin) [reply tools/snitch/snitcher/docker/linux/amd64/Dockerfile]: changed back to use 2 build stages.

File: tools/snitch/snitcher/docker/linux/amd64/Dockerfile (hunk 2 above)
Comment (philipktlin) [reply tools/snitch/snitcher/docker/linux/amd64/Dockerfile]: addressed by using matched openssl during build and run.

**Pull request merged**